next-cloudinary/src/components/CldOgImage/CldOgImage.tsx

```diff
@@ -1,29 +1,26 @@
 import React from 'react';
 import Head from 'next/head';
 import { constructCloudinaryUrl } from '@cloudinary-util/url-loader';
+import type { ImageOptions } from '@cloudinary-util/url-loader';
 
 import { NEXT_CLOUDINARY_ANALYTICS_ID, NEXT_CLOUDINARY_VERSION, NEXT_VERSION } from '../../constants/analytics';
+import { CldImageProps } from '../CldImage/CldImage';
 
 const IMAGE_WIDTH = 2400;
 const IMAGE_HEIGHT = 1200;
 
 const TWITTER_CARD = 'summary_large_image';
 
-export interface CldOgImageProps {
-  alt: string;
-  crop?: string;
+export type CldOgImageProps = Omit<CldImageProps, 'width, height' > & {
   excludeTags?: Array<string>;
-  gravity?: string;
-  height: string | number;
-  src: string;
   twitterTitle?: string;
-  width: string | number;
 }
 
 const CldOgImage = ({ excludeTags = [], twitterTitle, ...props }: CldOgImageProps) => {
-  const options = {
+  const { alt } = props;
+
+  const options: ImageOptions = {
     ...props,
-    alt: props.alt,
     crop: props.crop || 'fill',
     gravity: props.gravity || 'center',
     height: props.height || IMAGE_HEIGHT,
```

Comment:
height and width should probably not be removed from the incoming params definition since they are being referenced here? That way, user can override?

Reply:
yeah you're right. i think i got mixed up because it has default, but it still needs types